src/Avalonia.Input/DragEventArgs.cs

```diff
@@ -13,7 +13,7 @@ public class DragEventArgs : RoutedEventArgs
 
         public IDataObject Data { get; private set; }
 
-        public InputModifiers Modifiers { get; private set; }
+        public KeyModifiers Modifiers { get; private set; }
```

Comment:
Note that this is a technically a breaking change, so we should have an obsolete `Modifiers` with separate `KeyModifiers`. Especially with `KeyModifiers` name being used everywhere else.

Reply:
UWP actually includes both keyboard and mouse modifiers. I wonder what about people that want to check if their drag operation is being done using right mouse button?

https://docs.microsoft.com/en-us/uwp/api/windows.applicationmodel.datatransfer.dragdrop.dragdropmodifiers